.github/workflows/autodocs-enforce.yaml

```diff
@@ -103,6 +103,13 @@ jobs:
     - name: npm run build
       run: npm run build
 
+    - name: Set up Octo-STS
+      uses: chainguard-dev/octo-sts-action@6177b4481c00308b3839969c3eca88c96a91775f # v1.0.0
+      id: octo-sts
+      with:
+        scope: chainguard-dev/edu
+        identity: edu
```

Comment:
Im not sure the proper thing to do here, but I will share what I know about octo-sts..

The `identity` should match a file name in the `scope` repo, under `.github/chainguard`: https://github.com/chainguard-dev/edu/tree/main/.github/chainguard

So the fact that there isn't a `edu.sts.yaml` there means this would not work

Reply:
Makes sense, thanks for explaining how this works. I've updated this PR with an `edu.sts.yaml` file